benchmark/Streamly/Benchmark/Data/Array.hs

```diff
@@ -122,6 +123,8 @@ o_1_space_elimination value =
         , benchIOSink value "toStreamRev" Ops.toStreamRevDrain
         , benchFold "writeLastN.1" (S.fold (IA.writeLastN 1)) (P.sourceUnfoldrM value)
         , benchFold "writeLastN.10" (S.fold (IA.writeLastN 10)) (P.sourceUnfoldrM value)
+        , benchFold "writeLastN.Prim.1" (S.fold (IPA.writeLastN 1)) (P.sourceUnfoldrM value)
+        , benchFold "writeLastN.Prim.10" (S.fold (IPA.writeLastN 10)) (P.sourceUnfoldrM value)
```

Comment:
We can remove these two lines, instead just import `Data.Array as IA` when `DATA_ARRAY` cpp macro is defined.